unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SignupHUD/SignupHUDController.cs

```diff
@@ -12,17 +12,20 @@ public class SignupHUDController : IHUD
         internal BaseVariable<bool> signupVisible => DataStore.i.HUDs.signupVisible;
         internal IHUD avatarEditorHUD;
         private readonly NewUserExperienceAnalytics newUserExperienceAnalytics;
+        private DataStore_LoadingScreen loadingScreenDataStore;
 
         internal virtual ISignupHUDView CreateView() => SignupHUDView.CreateView();
 
         public SignupHUDController()
```

Comment:
Feel free to remove this constructor

Reply:
Im not sure how I can remove it. Im using this varaiable to desuscribe the listener on dispose

Reply:
Done!